README.md

```diff
@@ -12,6 +12,7 @@ This repository is licensed under [GNU Affero General Public License v3.0 (AGPL-
 
 pip install oedialect
 
-## Example
+## Tutorials
 
-You can find a basic example [here](doc/example/oedialect_basic_example.ipynb).
+You can find tutorials and examples [here](https://github
```

Comment:
Link broken (due to line break)

Reply:
Fixed in 9b9108b0faacb4bbc47f70674cd722737aecc01b